control/control.yaml

```diff
@@ -73,6 +73,23 @@ components:
           type: string
           enum: [start, stop]
 
+    Flows.Update:
+      description: A container of flows with associated property updates intended for on the fly change
```

Comment:
although we can mention "on the fly" in brackets, I think we should be more explicit by saying flow properties will be updated without affecting the transmit state